github_connector_odoo/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,11 @@
+* Once installed, go to your organization, and set extra settings:
+
+1. The name of your organization in the author keys of the manifest odoo
+   modules
+2. The URL of the file that contains IDs of your repositories for the runbot
+
+.. image:: /github_connector_odoo/static/description/github_organization_form.png
```

Comment:
This URL path is not correct. It should be relative as you can see here: https://github.com/OCA/maintainer-tools/blob/36453afeaed039f5f65e5cc4dd7910cb3235b811/tools/gen_addon_readme.py#L148